quisp/modules/QNIC/StationaryQubit/LogicalBellSteane713.txt

```diff
@@ -0,0 +1,16384 @@
+8.838834764831846047e-02 0.000000000000000000e+00
```

Comment:
could you remove this file?

Reply:
That file contains vector of logical qubit state that needed when we do a measurement for state-tomography. So we cannot remove that.

Reply:
ah here. I got it. 
https://github.com/sfc-aqua/quisp/pull/182/files#diff-268b7108d5342006616abc1242bfea24abcb899da935b1002706159301ceb7b2R255